### PR TITLE
persons: display idref persons

### PR DESCRIPTION
**Pull request by rerowep**
* Displays idref persons in detailed view.
* Updates documents fixtures to new mef with idref.

Co-Authored-by: Peter Weber <peter.weber@rero.ch>

## Why are you opening this PR?

- https://tree.taiga.io/project/rero21-reroils/task/1344

## How to test?

- Find persons with `IDREF`
- verify brief and detailed vue for persons from `IDREF`

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

**Comment by iGormilhit** [comment]
@rerowep maybe you could add a reference to MEF PR in the commit message: `This is related to rero/rero-mef#40.`

File: rero_ils/modules/persons/jsonschemas/persons/person-v0.0.1.json [hunk 1]
```diff
@@ -55,6 +56,10 @@
             {
               "label": "bnf",
               "value": "bnf"
+            },
+            {
+              "label": "idref",
```
Comment (iGormilhit) [inline rero_ils/modules/persons/jsonschemas/persons/person-v0.0.1.json:61]: The label should be `IdRef`

File: rero_ils/modules/persons/jsonschemas/persons/person-v0.0.1.json [hunk 2]
```diff
@@ -71,6 +76,10 @@
     "bnf": {
       "title": "BNF authorities",
       "type": "object"
+    },
+    "idref": {
+      "title": "IDREF authorities",
```
Comment (iGormilhit) [inline rero_ils/modules/persons/jsonschemas/persons/person-v0.0.1.json:81]: The title should be IdRef authorities.

Comment (rerowep) [reply rero_ils/modules/persons/jsonschemas/persons/person-v0.0.1.json]: changed

**Pull request merged**